clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -301,6 +301,11 @@ Changes in existing checks
   <clang-tidy/checks/modernize/use-designated-initializers>` check to fix a
   crash when a class is declared but not defined.
 
+- Improved :doc:`modernize-use-integer-sign-comparison
+  <clang-tidy/checks/modernize/use-integer-sign-comparison>` check to
+  add an option ``ConsideringIntegerSize``, that ignores a comparison between
```

Comment:
Consider something in this style:
Improved XYZ check to ignore comparisons between signed wide and unsigned narrow integer types, configurable via the `ConsideringIntegerSize` option.